lib/ansible/modules/reboot.py

```diff
@@ -68,9 +68,22 @@
       - Command to run that returns a unique string indicating the last time the system was booted.
       - Setting this to a command that has different output each time it is run will cause the task to fail.
     type: str
-    default: 'cat /proc/sys/kernel/random/boot_id'
+    default: '[determined based on target OS]'
```

Comment:
i would remove 'default' entry here as it is meaningless in the end, make note in the description that 'by default it will use a method that depends on teh target OS'